pkg/server/api_v2_sql_test.go

```diff
@@ -130,3 +130,36 @@ func getErrorResponse(t *testing.T, r []byte) (code, message string) {
 	require.NoError(t, err)
 	return er.Error.Code, er.Error.Message
 }
+
+func TestLocalityMetadataEnabledFilter(t *testing.T) {
```

Comment:
Can I request some additional tests executing these queries with the cluster setting off? Just to see that the result is what we expect. Or a loom showing what the console page looks like with the cluster setting on.